elementary/monitor/api/alerts/alerts.py

```diff
@@ -176,6 +169,41 @@ def _get_suppressed_alerts(
 
         return suppressed_alerts
 
+    def _get_latest_alerts(
+        self,
+        alerts: Union[
+            AlertsQueryResult[TestAlert],
+            AlertsQueryResult[ModelAlert],
+            AlertsQueryResult[SourceFreshnessAlert],
+        ],
+    ) -> List[str]:
+        alert_last_times = defaultdict(lambda: None)
+        latest_alert_ids = []
+        for alert in [*alerts.alerts, *alerts.malformed_alerts]:
+            id = (
+                alert.data.get("alert_class_id")
```

Comment:
What do you think about defining `__getattr__()` in `MalformedAlert` with `return self.data.get(...)`?

Reply:
Like!